RecoMTD/TrackExtender/plugins/TrackExtenderWithMTD.cc

```diff
@@ -306,6 +311,9 @@ namespace {
       tofpid.dterror =
```

Comment:
Minor comment, but it seems like you take the sqrt here, only to square this term several times below.  Consider simply using the square to begin with.